manifests/g/GiulioSorrentino/astroni/1.0/GiulioSorrentino.astroni.locale.it-IT.yaml

```diff
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.2.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: GiulioSorrentino.astroni
+PackageVersion: 1.0
+PackageLocale: it-IT
+Publisher: Giulio Sorrentino
+PublisherUrl: https://numerone.altervista.org
+Author: Giulio Sorrentino
+PackageName: numeronesoft background otto astroni
+PackageUrl: https://photos.google.com/share/AF1QipPURIPx4nopCKMligDjajXNVWLO1P_0kTHqJ6sGYVHTSs955arvwhjr281XzWBldA?key=aGpZR1oxYkpBXzcxbDRYYVYyalBCTWVtd0NUdTBB

```

Comment:
This link leads to a page of photos, and appears to be unrelated to the package.